core/models.py

```diff
@@ -1,5 +1,12 @@
+from django.core.validators import MaxValueValidator, MinValueValidator
 from django.db import models
 
+
+class Rating(models.Model):
+
+    rate = models.IntegerField(default=0, validators=[MaxValueValidator(5), MinValueValidator(0)])
```

Comment:
if we have default of 0 it is a super low rating, if we do a default of null we can then act on the fact it has not yet been set. Maybe can be changed in the next PR not a blocker for this one.

Just means if we add a cafe and not add a rating they automatically get the worst rating

Reply:
good point @lili2311, is it default=null default? or should I set it manually?